core/arch/arm/plat-vexpress/main.c

```diff
@@ -138,3 +138,71 @@ void console_flush(void)
 {
 	pl011_flush(CONSOLE_UART_BASE);
 }
+
+/*
+ * Adding MS specific property as a proof of concept
+ */
+
+#include <tee/tee_svc.h>
+#include <user_ta_header.h>
+#include <util.h>
+#include <kernel/tee_ta_manager.h>
+#include <kernel/tee_common_otp.h>
+#include <tee/tee_cryp_utl.h>
+
+/*
+ * The data to hash is 48 bytes made up of:
+ * - 16 bytes: the UUID of the calling TA.
+ * - 32 bytes: the hardware device ID
+ * The resulting endorsement seed is 32 bytes.
+ *
+ * The output buffer is the "binary" struct defined in
+ * the "prop_value" union and therefore comprises:
+ * -  4 bytes: the size of the binary value data (32)
+ * - 32 bytes: the binary value data (endorsement seed)
+ *
+ * Note that this code assumes an endorsement seed
+ * size == device ID size for convenience.
+ */
+static const size_t ta_endorsement_seed_size = 32;
```

Comment:
This doesn't have to be a global variable, a define or a local variable would be preferable.